src/components-examples/cdk/tree/cdk-tree-flat-children-accessor/cdk-tree-flat-children-accessor-example.ts

```diff
@@ -0,0 +1,55 @@
+import {ArrayDataSource} from '@angular/cdk/collections';
+import {CdkTree} from '@angular/cdk/tree';
+import {Component, ViewChild} from '@angular/core';
+import {timer} from 'rxjs';
+import {mapTo} from 'rxjs/operators';
+import {NestedFoodNode, NESTED_DATA} from '../tree-data';
+
+function* allNodes(nodes: NestedFoodNode[]): Iterable<NestedFoodNode> {
```

Comment:
Can we avoid creating a generator for the example? This would be too advanced to introduce in an example

Reply:
done.